app/src/main/java/ca/marshallasch/veil/FragmentCreatePost.java

```diff
@@ -104,7 +104,7 @@ private boolean submitPost() {
         String authorID = currentUser.getUuid();
 
 
-        ArrayList<String> tagList = (ArrayList<String>) Arrays.asList(tags.split(","));
+        ArrayList<String> tagList = new ArrayList<>(Arrays.asList(tags.split(",")));
```

Comment:
Add in a TODO comment to switch this out for categories later since that was the purposed solution.